eox_essence/interface/enrollment.py

```diff
@@ -0,0 +1,55 @@
+"""
+"""
+from django.contrib.auth.models import User
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from eox_essence.wrapper.edxapp.enrollments import api
+from eox_essence.wrapper.edxapp.student import models
+
+
+def get_enrollment(username, course_id, serialized=False):
```

Comment:
Since all this are functions in a module, there is no real enforcement on the functions and or function signatures that we can implement.

Should we create an abstract class and expose a class instead? if we do so and we require the APIs to always extend the AbstractEoxEssenceAPI we have a lot more control on the stability of the final API